src/Forms/ConfirmedPasswordField.php

```diff
@@ -723,6 +807,26 @@ public function getMinPasswordStrength(): int
         return $this->minPasswordStrength;
     }
 
+    /**
+     * Get the minimum password strength used for client-side evaluation
+     * where strength is defined on EntropyPasswordValidator
```

Comment:
```suggestion
     * Get the minimum password strength used for client-side evaluation.
     * If we're on a member form, the strength may come from the member password validator.
```
It's not _only_ defined on that validator, because if `getRequireStrongPassword()` is true and `getMinPasswordStrength()` is higher than `$validator->getRequiredStrength()` (or we're not on a member form or the member uses a different validator), then we use `getRequireStrongPassword()`

Reply:
Updated